opensrp-opd/src/main/java/org/smartregister/opd/activity/BaseOpdProfileActivity.java

```diff
@@ -112,17 +171,122 @@ public void setProfileID(@NonNull String registerId) {
 
     @Override
     public void setProfileAge(@NonNull String age) {
-        ageView.setText(String.format(getString(R.string.age_details), age));
+        genderView.setText(String.format(getString(R.string.age_details), age));
 
     }
 
     @Override
     public void setProfileGender(@NonNull String gender) {
-        genderView.setText(String.format(getString(R.string.gender_details), gender));
+        ageView.setText(gender);
```

Comment:
Yes & No, the current implementation of the layout on core does not enable use to change re-arrange the views, thus I just switched them here. Otherwise, we can use the overriden to change the arrangement of the views.

What do you think?

Reply:
Cool, whichever way you choose i will be okay with it